coala_quickstart/Extensions.py

```diff
@@ -0,0 +1,166 @@
+exts = {'.w': {'C'},
+        '.c': {'C'},
+        '.cs': {'C#'},
+        '.csx': {'C#'},
+        '.c++': {'C++'},
+        '.cpp': {'C++'},
+        '.cxx': {'C++'},
+        '.h': {'C', 'C++'},
+        '.h++': {'C++'},
+        '.hh': {'C++'},
+        '.hpp': {'C++'},
+        '.hxx': {'C++'},
+        '.tcc': {'C++'},
+        '.tpp': {'C++'},
+        '.cmake': {'CMake'},
+        '.cmake.in': {'CMake'},
+        '.css': {'CSS'},
+        '._coffee': {'CoffeeScript'},
+        '.coffee': {'CoffeeScript'},
+        '.cson': {'CoffeeScript'},
+        '.iced': {'CoffeeScript'},
+        '.cu': {'Cuda'},
+        '.cuh': {'Cuda'},
+        '.dart': {'Dart'},
+        '.f': {'FORTRAN'},
+        '.f03': {'FORTRAN'},
+        '.f08': {'FORTRAN'},
+        '.f77': {'FORTRAN'},
+        '.f90': {'FORTRAN'},
+        '.f95': {'FORTRAN'},
+        '.for': {'FORTRAN'},
+        '.fpp': {'FORTRAN'},
```

Comment:
It doesn't really matter internally since we `.lower()` everything, but it's called `Fortran` not `FORTRAN` FWIW. I changed it :+1: